Cargo.toml

```diff
@@ -20,34 +20,33 @@ maintenance = { status = "actively-developed" }
 
 [features]
 default = []
-google = ["base64", "google-pubsub1", "yup-oauth2"]
-mock = []
+google = ["base64", "yup-oauth2", "hyper", "http"]
 
 [[example]]
 name = "publish"
 required-features = ["google"]
 
 [dependencies]
-base64 = { version = "^0.10", optional = true }
-thiserror = "1"
-google-pubsub1 = { version = "^1.0.8", optional = true }
-# This project intentionally uses an old version of Hyper. See
-# https://github.com/Byron/google-apis-rs/issues/173 for more
-# information.
-hyper = "^0.10"
-hyper-rustls = "^0.6"
+futures = "0.3"
+serde_json = "^1"
 serde = { version = "^1.0", features = ["derive"] }
-serde_json = "^1.0"
-url = "^1.7"
+thiserror = "1"
+url = "2"
```

Comment:
I'm assuming this would pin to the major version?

Reply:
Yeah, it admits any version that’s major 2.